.github/pull_request_template.md

```diff
@@ -1,21 +1,23 @@
 # Description
 
-(description of your pull request here)
+(Description of your pull request goes here. **Provide examples and/or screenshots** unless the change is trivial)
 
-# Tests
+# Tests + Formatting
 
-Make sure you've done the following:
+Make sure you've done the following, if applicable:
 
-- [ ] Add tests that cover your changes, either in the command examples, the crate/tests folder, or in the /tests folder.
-- [ ] Try to think about corner cases and various ways how your changes could break. Cover them with tests.
-- [ ] If adding tests is not possible, please document in the PR body a minimal example with steps on how to reproduce so one can verify your change works.
+- [ ] Add tests that cover your changes (either in the command examples, the crate/tests folder, or in the /tests folder)
+  - Try to think about corner cases and various ways how your changes could break. Cover those in the tests
 
 Make sure you've run and fixed any issues with these commands:
 
 - [ ] `cargo fmt --all -- --check` to check standard code formatting (`cargo fmt --all` applies these changes)
 - [ ] `cargo clippy --workspace --features=extra -- -D warnings -D clippy::unwrap_used -A clippy::needless_collect` to check that you're using the standard code style
-- [ ] `cargo test --workspace --features=extra` to check that all the tests pass
```

Comment:
I am not sure if having the boxes to manually check really adds something but a distraction. CI will tell us anyways so maybe we just keep the commands for reference.

Reply:
Good call. Removed the checkboxes.